snuba_sdk/expressions.py

```diff
@@ -26,7 +26,7 @@ def validate(self) -> None:
 # For type hinting
 ScalarLiteralType = Union[None, bool, str, bytes, float, int, date, datetime]
 ScalarSequenceType = Sequence[Union[Expression, ScalarLiteralType]]
-ScalarType = Union[ScalarLiteralType, ScalarSequenceType]
+ScalarType = Union[ScalarLiteralType, ScalarSequenceType, Expression]
```

Comment:
We shouldn't need this any more, and if we do there's a deeper issue we should unravel.